docs/Changelog.md

```diff
@@ -26,6 +26,7 @@
 * [#6437](https://github.com/TouK/nussknacker/pull/6437) Removed deprecated operation to create a scenario:
   POST `/api/processes/{name}/{category}`. POST `/api/processes` should be used instead.
 * [#6415](https://github.com/TouK/nussknacker/pull/6415) Added "Component group" field to fragment properties, which allows selection of the group of components in the Creator Panel in which the fragment will be available
+* [#6213](https://github.com/TouK/nussknacker/pull/6213) Improvement: Load resource config only in test context
```

Comment:
There is something wrong with this explanation, I don't understand it :) TBH I don't know if it is worth to mention about this change to our users, this test-utils module is rather an internal thing, it generates only a noise for users, migration guide entry is enough for me.